docs/content/dagster-cloud/deployment/agents/running-multiple-agents.mdx

```diff
@@ -7,59 +7,89 @@ platform_type: "cloud"
+Each Dagster Cloud full deployment (e.g., `prod`) needs to have at least one agent running. A single agent is adequate for many use cases, but you may want to run multiple agents for the following reasons:
 
-If you want to distribute runs across multiple Kubernetes clusters, ECS clusters, EC2 instances, or other environments, you can run multiple agents. For example, one in each cluster, instance, etc.
+- Provide redundancy
+- Distribute runs across multiple Kubernetes clusters, ECS clusters, EC2 instances, or other environments
 
 It's recommended to only use multiple agents of the same type (e.g. multiple Kubernetes agents). Submitting runs to a specific agent isn't currently supported.
 
 ---
 
-## Prerequisites
+## Multiple agents in the same environments
 
-To complete the steps in this guide, you'll need:
+If you want to run multiple agents in the same environment (e.g., multiple Kubernetes agents in the same namespace), you can set the number of replicas in the configuration for your particular agent type:
 
-- **dagster-cloud version 0.13.0 or later for both agents and job code.**
-- **To use a Kubernetes, Docker, or local agent.** These agent types can support running multiple agents.
+### In Docker
 
----
+In docker, you can set the number of replicas for a service in the `docker-compose.yaml` file if the deployment mode is set to `replicated` (which is the default):
+
+```yaml
+services:
+  dagster-cloud-agent:
+    ...
+    deploy:
+      mode: replicated
+      replicas: 2
+```
 
-## Step 1: Set up the agent
+### In Kubernetes
 
-Follow the setup guide for your agent:
+In Kubernetes, the number of replicas is set in the Helm chart. You can set the number of replicas in the Helm command:
 
-- [Docker](/dagster-cloud/deployment/agents/docker)
-- [Kubernetes](/dagster-cloud/deployment/agents/kubernetes/configuring-running-kubernetes-agent)
-- [Local](/dagster-cloud/deployment/agents/local)
+```shell
+helm upgrade \
+    ...
+    --set replicas=2
+```
 
-After the agent is up and running, move on to the next step.
+or if using a `values.yaml` file:
+
+```yaml
+dagsterCloudAgent:
+  ...
+  replicas: 2
+```
+
+### In ECS
+
+In ECS, the number of replicas can be set via the cloudformation template:
+
+```yaml
+DagsterCloudAgent:
+  Type: AWS::ECS::Service
+  Properties:
+    ...
+    DesiredCount: 2
+```
+
+If using the cloudformation template provided by Dagster, the number of replicas can be set via the `NumReplicas` parameter in the AWS UI.
 
 ---
 
-## Step 2: Enable the agent_replicas setting
+## Multiple agents in different environments
 
-In this step, you'll enable the `agent_replicas` option to turn on running multiple agents. You can also choose to label the agent, making it easily identifiable on Dagster Cloud's **Status** page.
+<Note>
+  This feature requires dagster-cloud version 0.13.0 or later for both agents
+  and job code. Sending requests to specific agents is not supported.{" "}
+</Note>
```

Comment:
i think we can drop this at this point, 0.13.0 is ancient history (lower than the min version we say that we support elsewhere)